augur/curate/apply_geolocation_rules.py

```diff
@@ -220,16 +227,28 @@ def register_parser(parent_subparsers):
 def run(args, records):
     location_fields = [args.region_field, args.country_field, args.division_field, args.location_field]
 
-    geolocation_rules = load_geolocation_rules(args.geolocation_rules, args.case_sensitive)
+    with as_file("geolocation_rules.tsv") as default_rules_file:
+        default_rules = load_geolocation_rules(default_rules_file, args.case_sensitive)
 
-    for record in records:
+    custom_rules = {}
+    if args.geolocation_rules:
+        custom_rules = load_geolocation_rules(args.geolocation_rules, args.case_sensitive)
 
+    for record in records:
+        # First try to apply the custom rules provided by the user
         annotated_values = transform_geolocations(
-            geolocation_rules,
+            custom_rules,
             [record.get(field, '') for field in location_fields],
             args.case_sensitive)
 
+        # If no custom rules are applied, then try to apply the default rules
```

Comment:
[scanning through code] Is this the bit that resolves #1648?

Reply:
👍 I imagine you can still get circular-errors _within_ the custom rules, but this seems acceptable.

Reply:
Yeah, I'd consider circular-errors _within_ the custom rules a user error. Hmm, maybe we should make it easier to check for cyclic rules in the rules file...